.github/workflows/test-windup.yml

```diff
@@ -0,0 +1,57 @@
+name: Test Windup Addon
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+      - uses: actions/checkout@v3
+      - run: make fmt
+  vet:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+      - uses: actions/checkout@v3
+      - run: make vet
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+      - uses: actions/checkout@v3
+      - run: make cmd
+  build-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: make build-image
+  test-e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Minikube Cluster
+        uses: konveyor/tackle2-operator/.github/actions/start-minikube@main
+      - name: Build image in minikube
+        run: |
+          export SHELL=/bin/bash
+          eval $(minikube -p minikube docker-env)
+          make build-image
+      - name: Install Tackle 
+        uses: konveyor/tackle2-operator/.github/actions/install-tackle@main
+        with:
+          tackle-addon-windup-image: tackle2-addon-windup:latest
+#       - name: Forwarding Port for tackle-hub service
+#         run: |
+#           kubectl port-forward -n konveyor-tackle service/tackle-hub 8080:8080 &
+      - name: Make curl request to tackle-hub
+        run: |
+         echo "HOST=$(minikube ip)/hub" >> $GITHUB_ENV
+         chmod +x ./hack/task.sh
+         ./hack/task.sh
```

Comment:
Before running this to get the addon going, We need to set everything up by running [this](https://github.com/konveyor/tackle2-hub/blob/main/hack/add/all.sh) or something similar to it.